BitmarkRegistry/AppDelegate.swift

```diff
@@ -100,7 +100,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   }
 
   func application(_ application: UIApplication, didFailToRegisterForRemoteNotificationsWithError error: Error) {
-    registerAPNSSubject.onError(error)
+    #if !targetEnvironment(simulator)
+      registerAPNSSubject.onError(error)
```

Comment:
I'm ok with this :)
but just adding thought that: why don't we stop calling register a remote notification from the beginning; instead of calling and know that we will ignore the error anyway.

Reply:
Either is fine.